src/html/set-status.js

```diff
@@ -19,12 +20,16 @@ function setStatus({ response = {}, error }, { logger }) {
   // if there is an error, send a 500
   if (error) {
     logger.debug('payload.error -> 500');
-    return {
-      response: {
-        status: 500,
-        body: '',
-      },
-    };
+    const isDev = !production();
+    const res = {};
+    res.status = 500;
+    if (isDev) {
+      res.body = `<html><body><h1>500</h1><p>${error}</p></body></html>`;
+      res.headers = { 'Content-Type': 'text/html' };
+    } else {
+      res.body = '';
+    }
+    return res;
```

Comment:
you're missing a `response` object in your response :-)

```suggestion
    return {
       response: res,
    };
```

Reply:
Oops :)